client/jetpack-cloud/sections/agency-dashboard/sidebar/index.tsx

```diff
@@ -32,10 +32,21 @@ const DashboardSidebar: FunctionComponent< Props > = ( { path } ) => {
 
 	const isPluginsPage = path.includes( '/plugins' );
 	const isPluginManagementEnabled = config.isEnabled( 'jetpack/plugin-management' );
+	const isWPCOMAtomicSiteCreationEnabled = config.isEnabled(
+		'jetpack/pro-dashboard-wpcom-atomic-hosting'
+	);
 
 	return (
 		<div>
-			<SiteSelector showAddNewSite showAllSites allSitesPath={ path } siteBasePath="/backup" />
+			<SiteSelector
+				showAddNewSite
+				showAllSites
+				allSitesPath={ path }
+				siteBasePath="/backup"
+				wpcomSiteBasePath={
+					isWPCOMAtomicSiteCreationEnabled ? 'https://wordpress.com/home' : false
```

Comment:
We are making this value string or boolean. WDYT of changing this to 

```suggestion
					isWPCOMAtomicSiteCreationEnabled && 'https://wordpress.com/home'
```

or

```suggestion
					isWPCOMAtomicSiteCreationEnabled ? 'https://wordpress.com/home': ''
```

Reply:
Good suggestions 👍. I updated the code using your first suggestion.